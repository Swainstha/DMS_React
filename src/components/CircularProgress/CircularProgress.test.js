import React from 'react';
import CircularProgress from './index';
import {
    configure,
    mount,
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})


const setup = (props = {}) => {
    return mount( < CircularProgress {...props}  />
    )
}


it("CircularProgress",()=>{
    const wrapper= setup();
    expect(wrapper).toMatchSnapshot();
})