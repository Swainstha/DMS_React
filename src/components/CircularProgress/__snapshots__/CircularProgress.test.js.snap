// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CircularProgress 1`] = `ReactWrapper {}`;
